pkg/hostagent/pods.go

```diff
@@ -472,16 +472,16 @@ func (agent *HostAgent) syncEps() bool {
 			if seen[ep.Uuid] {
 				continue
 			}
+			epfile := agent.FormEPFilePath(ep.Uuid)
 			poduuid := strings.Split(ep.Uuid, "_")[0]
-			ep.SnatUuid, err = agent.getSnatUuids(poduuid)
+			ep.SnatUuid, err = agent.getSnatUuids(poduuid, epfile)
```

Comment:
Isn't this the case where the EP file does not exist and we are creating a new one? If so, I guess we should not return error in getSnatUuids() if the file is not present...

Reply:
addressed